templates/tower-project.yaml

```diff
@@ -46,8 +46,8 @@ Parameters:
     Type: Number
     Description: >-
       Number of days until S3 objects in the short-term (scratch) bucket are
-      moved to ScratchLifecycleStorageClass. Default is 30 days.
-    Default: 30
+      moved to ScratchLifecycleStorageClass. Default is 14 days.
+    Default: 14
```

Comment:
This should change to 10 or 20

Reply:
@BrunoGrandePhD I think I remember that as well... I changed it back to 30 days for now

Reply:
This parameter has to do with the transition to intelligent tiering in S3. It's unrelated to data expiration.